prime-router/src/main/kotlin/azure/ProcessFunction.kt

```diff
@@ -0,0 +1,40 @@
+package gov.cdc.prime.router.azure
+
+import com.microsoft.azure.functions.ExecutionContext
+import com.microsoft.azure.functions.annotation.FunctionName
+import com.microsoft.azure.functions.annotation.QueueTrigger
+import com.microsoft.azure.functions.annotation.StorageAccount
+import java.util.logging.Level
+
+const val process = "process"
```

Comment:
I don't like this too much.  This const has two meanings: the name of the function and the name of the queue.  I would say that if you have to use constants then make two of them.

Reply:
Lets discuss - could easily change here, right now it follows the pattern used in both BatchFunction and SendFunction

Reply:
Updated to split into two variable. This no longer matches our BatchFunction or SendFunction structure but is more clear.